artifactory_test.go

```diff
@@ -224,6 +224,21 @@ func TestArtifactorySimpleUploadSpecUsingConfig(t *testing.T) {
 	inttestutils.VerifyExistInArtifactory(tests.GetSimpleUploadExpectedRepo1(), searchFilePath, serverDetails, t)
 	cleanArtifactoryTest()
 }
+func TestReleaseBundleImportOnPrem(t *testing.T) {
+	initArtifactoryTest(t, "")
+	initLifecycleCli()
+	// Sets the public key in Artifactory to accept the signed release bundle.
+	sendArtifactoryTrustedPublicKey(artHttpDetails)
+	// Import the release bundle
+	wd, err := os.Getwd()
+	assert.NoError(t, err)
+	testFilePath := filepath.Join(wd, "testdata", "lifecycle", "import", "cli-tests-2.zip")
+	assert.NoError(t, lcCli.Exec("rbi", testFilePath))
+
+	// Cleanup
+	deleteReceivedReleaseBundle("cli-tests", "2")
+	cleanArtifactoryTest()
```

Comment:
Best to `defer cleanArtifactoryTest()` in the beginning of the test in case we do not reach the end of the function.